src/components/PostList/PostList.jsx

```diff
@@ -1 +1,9 @@
-export const PostList = () => <>Put the list here</>;
+import { PostInfo } from '../PostInfo/PostInfo';
+
+export const PostList = ({ posts }) => (
```

Comment:
Ensure that the `posts` prop is passed correctly to the `PostList` component. It should be an array of post objects, each with an `id` property.